src/list-data/entity-type-data.ts

```diff
@@ -281,6 +281,177 @@ export const EntityTypesBcData: EntityI[] = [
   }
 ]
 
+/** List of entity types for Continuation In NRs. */
```

Comment:
This is almost the same as the EntityTypesBcData, above, except with only the 6 cont in entity types. The category and blurbs are the same.

(Yes, cont in coop is still CP. This is an ongoing discussion and may be fixed in a separate ticket.)